tests/getprop.awk

```diff
@@ -1,3 +1,28 @@
+# Extract test property values from CTestTestfile.cmake
+#
+# User must initialise two awk variables, typically through the "-v" option,
+# when invoking this script
+#
+#   search: Search pattern.  Typically a string such as
+#      set_tests_properties($test_name
+#
+#   prop: Property name, for instance DIRNAME or SIMULATOR.
+#
+# Property value will be printed to the standard output stream.
+#
+# The script assumes that $1 on candidate lines is suitable for matching
+# against 'search', and that $2 of the mathcing lines is the word
```

Comment:
matching

Reply:
> matching

Duh, thanks.  I've pushed an update to fix it.